canonical-web-and-design/ubuntu.com
# Refactor release cycle page into tabbed content

## Done

- Refactored /about/release-cycle content into tabs, according to the [copy doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11162.demos.haus/about/release-cycle
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the content of the page is now in tabs, and that the tabs work as expected
- Visit:
  - https://ubuntu-com-11162.demos.haus/about/release-cycle#ubuntu
  - https://ubuntu-com-11162.demos.haus/about/release-cycle#ubuntu-kernel-release-cycle
  - https://ubuntu-com-11162.demos.haus/about/release-cycle#openstack-release-cycle
  - https://ubuntu-com-11162.demos.haus/about/release-cycle#kubernetes-release-cycle
- See that the correct tabs are displayed

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4746
